dbms/src/TiDB/Schema/SchemaBuilder.cpp

```diff
@@ -715,31 +716,32 @@ void SchemaBuilder<Getter, NameMapper>::applyPartitionDiff(const TiDB::DBInfoPtr
 }
 
 template <typename Getter, typename NameMapper>
-void SchemaBuilder<Getter, NameMapper>::applyRenameTable(const DBInfoPtr & new_db_info, TableID table_id)
+void SchemaBuilder<Getter, NameMapper>::applyRenameTable(const DBInfoPtr & new_db_info, TableID table_id, std::string_view action)
 {
```

Comment:
Maybe use enum instead of `string_view`?

Reply:
No. I think string_view is better than enum. Because something I attach more info than just a enum name. For example, attaching the table_id
https://github.com/pingcap/tiflash/blob/5b9d758e1d4821795bd0cd6accd7ce24eef6a7c2/dbms/src/TiDB/Schema/SchemaBuilder.cpp#L682-L683
https://github.com/pingcap/tiflash/blob/5b9d758e1d4821795bd0cd6accd7ce24eef6a7c2/dbms/src/TiDB/Schema/SchemaBuilder.cpp#L1241-L1245